LEGO1/pizza.cpp

```diff
@@ -1,13 +1,35 @@
 #include "pizza.h"
 
-// OFFSET: LEGO1 0x10037ef0 STUB
+// OFFSET: LEGO1 0x10037ef0
 Pizza::Pizza()
 {
-  // TODO
+  // FIXME: This inherits from LegoActor, probably why this isn't matching
+  this->m_unk80 = 0;
+  this->m_unk84 = 0;
+  this->m_unk88 = 0;
+  this->m_unk8c = -1;
+  this->m_unk98 = 0;
+  this->m_unk90 = 0x80000000;
 }
 
-// OFFSET: LEGO1 0x10038100 STUB
+// OFFSET: LEGO1 0x1002c7cf STUB
+// MxS32 IsleActor::Notify(MxParam *)
+// {
+//   // TODO
+//   return 0;
+// }
```

Comment:
Shouldn't this stub be moved to `isleactor.cpp`?

Reply:
Good point, in matrix @ktkaufman03 suggested it, I don't know why I didn't move it over there instead of stubbing it here in the wrong file